L1Trigger/TrackerTFP/interface/DataFormats.h

```diff
@@ -0,0 +1,426 @@
+#ifndef L1Trigger_TrackerTFP_DataFormats_h
+#define L1Trigger_TrackerTFP_DataFormats_h
+
+#include "FWCore/Framework/interface/data_default_record_trait.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "L1Trigger/TrackerTFP/interface/DataFormatsRcd.h"
+#include "L1Trigger/TrackerDTC/interface/Setup.h"
+
```

Comment:
Add '#include "DataFormats/L1TrackTrigger/interface/TTBV.h' to make it clear where the TTBV used in this class is coming from.

Reply:
done